pkgs/development/python-modules/ducc0/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchFromGitLab, pythonOlder, pytestCheckHook, pybind11, numpy }:
+
+buildPythonPackage rec {
+  pname = "ducc0";
+  version = "0.23.0";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.mpcdf.mpg.de";
+    owner = "mtr";
+    repo = "ducc";
+    rev = "ducc0_" + builtins.replaceStrings ["."] ["_"] version;
```

Comment:
@fabaff thanks for your helpful comments. Apart from integrating these, I also updated this line here. I hope it is fine.

Reply:
```suggestion
    rev = "ducc0_${lib.replaceStrings ["."] ["_"] version}";
```